.github/workflows/enos-run-k8s.yml

```diff
@@ -42,6 +42,8 @@ jobs:
         uses: hashicorp/action-setup-enos@v1
         with:
           github-token: ${{ secrets.ELEVATED_GITHUB_TOKEN }}
+          # TODO: remove the version pin. This is only use to test a version that is not yet default.
```

Comment:
Do we want to remove these before merge, once we're done testing?

Reply:
Yeah, I'm testing enos 0.0.20 but haven't released that version to homebrew or to action-setup-enos yet. Once I'm convinced there are no bugs I'll release it in those channels, remove these TODOs and pins and then we'll merge this with the ent equivalent.